medicines/doc-index-updater/src/create_manager/mod.rs

```diff
@@ -84,11 +93,19 @@ pub async fn process_message(message: CreateMessage) -> Result<Uuid, anyhow::Err
     let metadata: BlobMetadata = message.document.into();
     let blob = create_blob(&storage_client, &file, metadata.clone().into()).await?;
     let name = blob.name.clone();
-    tracing::info!("Uploaded blob {} for job {}", &name, &message.job_id);
-
+    
+    tracing::info!(
```

Comment:
I'd be tempted to make this info log & the one above both debug logs tbh as they're not particularly notable events I would say? Dunno what other people think?